### PR TITLE
fix registration pending case with IDV

### PR DESCRIPTION
**Pull request by bhoggard**
follow-up for https://artsyproduct.atlassian.net/browse/AUCT-940 to fix pending registration state

### CODE REVIEW

File: src/desktop/apps/auction/components/layout/auction_info/Registration.js [hunk 1]
```diff
@@ -5,6 +5,33 @@ import { get } from "lodash"
...
+    defaultText = "Registration required to bid",
+    defaultColor = "black60",
+  } = props
+  if (userNeedsIdentityVerification) {
+    return (
+      <Sans mt="1" size="3" color="black60" textAlign="center">
+        Identity verification required to bid.{" "}
+        <a
+          target="_blank"
+          href="https://www.artsy.net/identity-verification-faq"
```
Comment (dleve123) [inline src/desktop/apps/auction/components/layout/auction_info/Registration.js:20]: Minor: Can scope this to the force environment that the user is in? 

```suggestion
          href="/identity-verification-faq"
```

We do something similar below:

https://github.com/artsy/force/blob/06a319844b18be07492b5494c4581594d9ca8fc9/src/desktop/apps/auction/components/layout/auction_info/Registration.js#L111

**Comment by yuki24** [comment]
I'm actually back-filling the test for this specific component as part of my PR and feel pretty good about this change, so I'll just go ahead and merge it.

**Comment by dleve123** [comment]
Re https://github.com/artsy/force/pull/5417#issuecomment-613662326

@yuki24 Considering you merged this, would you mind fixing the minor href issue mentioned in https://github.com/artsy/force/pull/5417#discussion_r408401950 in your testing PR?

**Comment by yuki24** [comment]
As a reference, here is a commit that brings back the test for this component: https://github.com/artsy/force/pull/5392/commits/b2307c5243916f9b211cd81942a113493482021b

@dleve123 It does make sense, but I also don't think it is extremely important or a blocker either... I've once lost the staging change in `/identity-verification-faq` because I wasn't using production as a single source of truth, so I could see pointing to prod useful as well.

**Comment by dleve123** [comment]
> @dleve123 It does make sense, but I also don't think it is extremely important or a blocker either... I've once lost the staging change in /identity-verification-faq because I wasn't using production as a single source of truth, so I could see pointing to prod useful as well.

IMHO: It's a small quality of life thing – it's a surprising gotcha whenever you jump from one environment to the other unintentionally. Is it a pressing problem? No, but, is it a wrinkle that is valuable to iron out once identified, yes.

>  lost the staging change in /identity-verification-faq because I wasn't using production as a single source of truth, so I could see pointing to prod useful as well.

Can you elaborate / be more concrete here? Not following you!

---

Generally, I see code review as a more than just removing blockers. I see it as a process for elevating quality of code (and a means for learning -- for both the reviewer and author) whenever (pragmatically) possible.

**Pull request merged**